<?xml version="1.0"?>

<launch>

    <arg name="camera"  default="mynteye"/>

    <arg name="is_rect" default="false"/>

    <arg name="yaml_charuco" default="$(find capture_cam)launch/config/capture_stereo_charuco.yaml"/>
    <arg name="yaml_checker" default="$(find capture_cam)launch/config/capture_stereo_checker.yaml"/>

    <node pkg="nodelet" type="nodelet" name="nodelet_manager"  args="manager" output="screen"/>

    <node pkg="nodelet" type="nodelet" name="capture_stereo"
          args="load capture_cam/CaptureCamStereo nodelet_manager">

        <remap from="left/image"        to="$(arg camera)/left/image_raw"/>
        <remap from="right/image"       to="$(arg camera)/right/image_raw"/>

        <param name="param_dir"  type="string" value="$(find capture_cam)/params_out"/>
        <param name="is_rectify" type="bool"   value="$(arg is_rect)"/>
        <rosparam file="$(arg yaml_checker)" command="load"/>

    </node>

</launch>
